### PR TITLE
MGMT-19331: Add `secure_boot_state` to inventory

### PR DESCRIPTION
**Pull request by jhernand**
This patch changes the agent so that it checks if secure boot is enabled and adds that to the inventory. To do the check it uses reads the state from the `/sys/firmware/efi/efivars/SecureBoot-...` file.

Related: https://issues.redhat.com/browse/MGMT-19331
Related: https://issues.redhat.com/browse/MGMT-19056

### CODE REVIEW

File: src/commands/actions/inventory_cmd.go [hunk 1]
```diff
@@ -62,10 +72,40 @@ func (a *inventory) Args() []string {
...
+	// unconditionally, as that will make the podman command fail.
+	const efivarsPath = "/sys/firmware/efi/efivars"
+	efivarsLogger := logrus.WithFields(logrus.Fields{
+		"path": efivarsPath,
+	})
+	_, err := a.filesystem.Stat(efivarsPath)
+	if os.IsNotExist(err) {
+		efivarsLogger.WithFields(logrus.Fields{
+			"path": efivarsPath,
+		}).Info("EFI variables filesystem isn't mounted")
```
Comment (adriengentil) [inline src/commands/actions/inventory_cmd.go:87]:
```suggestion
		efivarsLogger.Info("EFI variables filesystem isn't mounted")
```
`efivarsLogger` already contains this field

**Review by adriengentil — COMMENTED** [review]
Looks great just a small comment

File: src/commands/actions/inventory_cmd.go (hunk 1 above)
Comment (jhernand) [reply src/commands/actions/inventory_cmd.go:87]: Indeed, thanks @adriengentil.

Comment (jhernand) [reply src/commands/actions/inventory_cmd.go]: Done.

**Review by adriengentil — APPROVED** [review]

**Pull request merged**